### PR TITLE
Add ProjectId to Command model

### LINKED ISSUE #1057: Add check if requested command belongs to logged-in project, after adding project id field to model.Command.
https://github.com/pipe-cd/pipe/blob/f23db23ab0f1eb20cbe39cf349bac1c00a8f543a/pkg/app/api/api/web_api.go#L1115-L1118

### LINKED ISSUE #1575: Add ProjectId field to Command model
**What would you like to be added**:

**Why is this needed**:

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1057
Fixes #1575

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/model/command.proto [hunk 1]
```diff
@@ -61,9 +61,10 @@ message Command {
     string id = 1 [(validate.rules).string.min_len = 1];
     string piped_id = 2 [(validate.rules).string.min_len = 1];
     string application_id = 3;
-    string deployment_id = 4;
-    string stage_id = 5;
-    string commander = 6;
+    string project_id = 4;
+    string deployment_id = 5;
+    string stage_id = 6;
+    string commander = 7;
```
Comment (nghialv) [inline pkg/model/command.proto:67]: We have to keep the field numbers to ensure backward-compatibility.
The data with the old structure may be still in the cache or somewhere.

Comment (khanhtc1202) [reply pkg/model/command.proto:67]: Nice catch 👍 And besides, should it be `string project_id = 7;` 👀

Comment (khanhtc1202) [reply pkg/model/command.proto]: addressed on https://github.com/pipe-cd/pipe/pull/1772/commits/df9a46b82333b2059064f9b8b42fac038c4406a4 🙏

**Comment by khanhtc1202** [comment]
Let's me merge this one 🙏 
/approve

**Comment by nakabonne** [comment]
I'm about to have a look, but too late. Thank you for working on this long-lived issue!

**Comment by khanhtc1202** [comment]
> I'm about to have a look, but too late. Thank you for working on this long-lived issue!

Thank you 😜

**Pull request merged**